.github/ISSUE_TEMPLATE/documentation_improvement.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Documentation improvement
+about: Request a documentation improvement
+title: ''
+labels: 'C: Documentation'
+assignees: ''
+
```

Comment:
Is this empty line needed?

Reply:
@jougs Well-noticed! I added it for consistency with our other issue templates: https://github.com/nest/nest-simulator/tree/master/.github/ISSUE_TEMPLATE